### PR TITLE
Enable PHP 7.1 testing on Travis CI

### PR DESCRIPTION
**Pull request by asgrim**
Wasn't sure to base this off `master` or `develop`, but happy to rebase either way.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,6 +46,16 @@ matrix:
     - php: 7
       env:
         - DEPS=latest
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - DEPS=locked
+        - CHECK_CS=true
```
Comment (michalbundyra) [inline .travis.yml]: Probably we don't need to run CS check on PHP7.1, usually we run it only on one version, and as it is on PHP7 here is redundant.

Comment (asgrim) [reply .travis.yml]: :+1: removed

**Comment by Xerkus** [comment]
@asgrim rebase to master

**Comment by asgrim** [comment]
@Xerkus thanks, rebased to master

**Pull request merged**